README.md

```diff
@@ -8,6 +8,15 @@ The only differences with a traditional create-react-app application are :
 
  Note that you should make sure your server serves `.wasm` files with the right mimetype, that is: `application/wasm`. Otherwise, you'll see the following error: `TypeError: Response has unsupported MIME type`
  
- See [`src/App.js`](./src/App.js) for the code.
+Webpack 5 do not include nodejs polyfills by default, so you'll need to include them by adding them as dev dependency
```

Comment:
Can we add a link to https://github.com/sql-js/react-sqljs-demo/tree/v1 as an example of how to set things up for webpack 4 ?